tests/Unit/Evolution/Systems/Cce/AnalyticSolutions/RotatingSchwarzschild.py

```diff
@@ -0,0 +1,29 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def spherical_metric(sin_theta, cos_theta, t, r, m, f):
+    return np.array([[
```

Comment:
When you wrap mathematical expressions in parens `()` then yapf can often format them more nicely, since Python allows line breaks. Not sure about this particular case though, because `np.array` already provides parens.